DuckDuckGo/Updates/UpdateController.swift

```diff
@@ -71,7 +71,7 @@ final class UpdateController: NSObject, UpdateControllerProtocol {
         didSet {
             if let cachedUpdateResult {
                 latestUpdate = Update(appcastItem: cachedUpdateResult.item, isInstalled: cachedUpdateResult.isInstalled)
-                hasPendingUpdate = latestUpdate?.isInstalled == false && updateProgress.isIdle
+                hasPendingUpdate = latestUpdate?.isInstalled == false && updateProgress.isDone && userDriver?.isResumable == true
```

Comment:
Just optional: It would help with readability if this is wrapped up to a computed property and documented.